app/controllers/bars_controller.rb

```diff
@@ -18,6 +23,43 @@ def create
     end
   end
 
+  def show
+    @bar = Bar.find(params[:id])
+    @current_user = current_user
```

Comment:
You can just call `current_user` in your view instead of `@current_user` to kill this line in your controller.